setup.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
I think `/usr/bin/env bash` is more robust to `bash` being in different locations.  What's the reasoning behind this change?